### PR TITLE
add quick install script

### PR DESCRIPTION
**Pull request by gouzil**
add quick install script

### CODE REVIEW

**Comment by jzhang533** [comment]
Paddle do have a env collecting script:
https://github.com/PaddlePaddle/Paddle/blob/develop/tools/summary_env.py

but still lacks lots of env information, compared with pytorch's script:
https://github.com/pytorch/pytorch/blob/master/torch/utils/collect_env.py

May be it's a good chance to improve this script.

File: rfcs/Installation/20220830_quick_install_script.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+# 二、飞桨现状
+
+飞桨目前支持[环境检测](https://www.paddlepaddle.org.cn/documentation/docs/zh/install/pip/linux-pip.html#sanyanzhenganzhuang)和[快速安装脚本](https://fast-install.bj.bcebos.com/fast_install.sh)。但是并不会帮助用户安装前期的环境配置。
+
+# 三、业内方案调研
+
+TensorFlow 通过发布 [arch包](https://archlinux.org/packages/community/x86_64/python-tensorflow-opt-cuda/) 实现 (仅支持 arch linux), 通过 [conda](https://docs.anaconda.com/anaconda/user-guide/tasks/tensorflow/#cuda-versions) 实现自动安装 CUDA 和 cuDNN (支持windowns 和 linux)
+
+PyTorch 社区开发者通过发布 [arch包](https://aur.archlinux.org/packages/python-pytorch-opt-rocm?all_deps=1#pkgdeps) 实现 (仅支持 arch linux)
```
Comment (dingjiaweiww) [inline rfcs/Installation/20220830_quick_install_script.md:32]: It is advised to add the installation command of each solution and personal assessment of the entire install experience. please describe bugs if you occur

Comment (gouzil) [reply rfcs/Installation/20220830_quick_install_script.md]: Done add the installation command

**Comment by gouzil** [comment]
> Paddle do have a env collecting script: https://github.com/PaddlePaddle/Paddle/blob/develop/tools/summary_env.py
> 
> but still lacks lots of env information, compared with pytorch's script: https://github.com/pytorch/pytorch/blob/master/torch/utils/collect_env.py
> 
> May be it's a good chance to improve this script.

Compared to Paddle and PyTorch , TensorFlow report is more detailed, But it really gives a bit too much

**Comment by gouzil** [comment]
Add a picture that passed the test

截屏2022-10-09 上午10 31 07

**Comment by gouzil** [comment]
paddle-docs [pr](https://github.com/PaddlePaddle/docs/pull/5340) update

**Pull request closed without merging**